drivers/hmis_external_apis/app/models/hmis_external_apis/external_forms/config.rb

```diff
@@ -5,24 +5,24 @@
 ###
 
 module HmisExternalApis::ExternalForms
-  Config = Struct.new(:site_title, :site_logo_url, :site_logo_dimensions, :recaptcha_key, :presign_url, keyword_init: true) do
+  class Config
+    attr_reader :site_title, :site_logo_url, :site_logo_dimensions, :recaptcha_key, :presign_url, :csp_content
+
+    def initialize
+      # This could be dryed up if we repeat this pattern
+      attrs = [:site_title, :site_logo_url, :site_logo_dimensions, :recaptcha_key, :presign_url, :csp_content]
+      attr_keys = attrs.map { |attr| "external_forms/#{attr}" }
```

Comment:
This is probably a better way to do it, but I noticed you made the value column a `jsonb`, so you _could_ store the config as one key in a hash.  Do you think we'll ever want more than one `external_forms` implementation?  Would it make sense to add another variable to the key name to specify _which_ external form this is for?

Reply:
+1 We will definitely need to support multiple external forms with different configurations.

I wondered too about using a prefix `external_forms/` approach as opposed to just one key with json.

Reply:
If we need to support multiple external forms, I guess I'd lean on renaming this file to represent which form it backs, something like `intake.rb` and then the keys could be `external_forms/intake/...`.  Just a thought though, happy to entertain other ideas.

Reply:
I was wondering about `site_title` specifically - is that like the HTML title? i would think it should match the name of the form, rather than being the same across all sites.

Reply:
Yes, we currently use `site_title` as the HTML title. That's a good point, I agree it would be better to use the form definition title; it should be easy to adjust the layout to make that change. However we shouldn't need form-specific config properties for that since we can just use the definition.

Reply:
Ok, that makes sense.